api/v1beta1/common_types.go

```diff
@@ -264,4 +268,11 @@ const (
 	SecurityProtocolSaslSSL SecurityProtocol = "sasl_ssl"
 	// SecurityProtocolSaslPlaintext
 	SecurityProtocolSaslPlaintext SecurityProtocol = "sasl_plaintext"
+
+	// SSLClientAuthRequired states that the client authentication is required when SSL is enabled
+	SSLClientAuthRequired SSLClientAuthentication = "required"
+	// SSLClientAuthRequested states that the client authentication is requested, but a client without certs can still connect when SSL is enabled
+	SSLClientAuthRequested SSLClientAuthentication = "required"
```

Comment:
```suggestion
	SSLClientAuthRequested SSLClientAuthentication = "requested"
```

Reply:
Wasn't sufficiently caffeinated when I copied and pasted :sweat_smile:

I removed the `requested` and `none` consts since they are actually not being used